### ISSUE TITLE
bug with has_input in form?

### ISSUE THREAD

**Issue by kltm** [geneontology/simple-annoton-editor#71]
_From @pfey03 on May 21, 2018 16:30_

Hello,
I for the first time used the form to start a model to check it out. But when trying to add an affected gene/protein as has_input, it autofilled 'H2Bv3 Ddis' but wouldn't save (save not active and said the has_input is missing). Then I pasted the gene ID instead and I could save. However, when opening as the model this annotation was missing. I added it in the model  then.
gomodel:5ae3b0f600000395

Thanks,
Petra

_Copied from original issue: geneontology/noctua#560_

**Comment by kltm** [comment]
@pfey03 
Is it correct to say that this was in the Graph Editor, or the Form interface?

**Comment by kltm** [comment]
_From @pfey03 on May 21, 2018 18:16_

It was in the form that it didn't take it.
First, with autofill when I added the gene name 'H2Bv3 Ddis', the 'Save' button was not active and it said the has_input entry is missing, though it wasn't. Then I replaced it with our gene ID, but there was no obvious recognition of the ID, but then I could save! However, when opening the annotations in the graph editor this one was missing. I then added it in the graph editor.
